### PR TITLE
Provide page objects for easier overriding

### PR DESCRIPTION
**Pull request by Gallaecio**

### CODE REVIEW

**Comment by Gallaecio** [comment]
@kmike Shall I go ahead with this approach for all other types?

**Review by wRAR — APPROVED** [review]

File: tests/test_pages.py [hunk 1]
```diff
@@ -155,16 +155,17 @@ def validate_input(self):
 
 
 def test_page_pairs():
-    """For every page a base page, for every base page a page."""
+    """For every page a base page and an auto page, and the other way around."""
```
Comment (BurnzZ) [inline tests/test_pages.py:158]: Not sure if this is the right sentence fix.

```suggestion
    """For every base page and an auto page, and the other way around."""
```

**Review by BurnzZ — APPROVED** [review]

File: tests/test_pages.py (hunk 1 above)
Comment (Gallaecio) [reply tests/test_pages.py:158]: The test checks that for every base page there is both a matching page and a matching auto page, that for every page there is a matching base page, and that for every auto page there is a matching base page.

I think my proposed wording is the shortest to express that. But maybe my first paragraph in this comment would be a better docstring?

Comment (BurnzZ) [reply tests/test_pages.py:158]: How about `"""For every page, a base page and an auto page. Vice versa."""`. I think I'm confused with `"... page a base page ..."

**Review by kmike — APPROVED** [review]

**Pull request merged**